### PR TITLE
April Updates

### PR DESCRIPTION
**Pull request by rachelcgrant**
Changes proposed in this pull request:

* Created new section for the review forecast
* Added new gifs and descriptions to the Lessons and Reviews pages that match new dashboard images
* Edited ordering of the onboarding series so that the review forecast came in 7th in the sequence
* Edited a typo on the Small vs Large characters page

---

The credentials to view the Netlify deploy is the following:

* **username**: crabigator
* **password**: googlebegone

### CODE REVIEW

**Comment by vietqhoang** [comment]
Deploy preview for *wanikani-knowledgebase* processing.

Building with commit ba31bc52fbee0923001a18c0a110c0fb5791c773

https://app.netlify.com/sites/wanikani-knowledgebase/deploys/5e9a2f901be4c6000606edc7

File: _posts/2020-04-17-review-forecast.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+type: Document
+---
+
+Since you're now up-to-date with *how* reviews work, soon you'll be able to figure out *when* you'll be getting them through the Review Forecast.
+
+The Review Forecast is located on the righthand side of the dashboard. It's the set of boxes with numbers and bars that indicates how many review items you have coming up.
+
+![WaniKani review forecast](/images/review-forecast.png)
+
+Looking at this, you can see that the number of reviews you have *today* are shown first. Hovering your mouse over the different numbers shows more information. The forecast is read horizontally and divided into hourly categories. The number to the right of the "+" sign tells you how many new reviews you will get on the hour. The green bar also reflects this number. The number to on the far right side shows you how many reviews you will have in total. You can see that it increases by increments of each new number of hourly reviews.
```
Comment (JenKay) [inline _posts/2020-04-17-review-forecast.md:19]: "The number to on the far" <-- delete 'to'

File: _posts/2020-04-17-review-forecast.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+date: 2020-04-17 10:00
+title: Review Forecast
+categories:
+  - getting-started
+  - wanikani
+description:
+set: onboarding
```
Comment (JenKay) [inline _posts/2020-04-17-review-forecast.md:8]: It looks like this post is showing up as number 1 on the Onboarding series: 
image
Should we move it to either before or after '4. why do i need to wait'?

File: _posts/2018-10-31-first-lessons.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ set-order: 2
 type: Document
 ---
 
-Now that you have a basic understanding of [how WaniKani works](/getting-started/how-wanikani-works/), it's time to learn something. To begin, hit the "Lessons" button in the top left corner of your screen.
+Now that you have a basic understanding of [how WaniKani works](/getting-started/how-wanikani-works/), it's time to learn something. To begin, hit the pink "Lessons" button that's located below the "WaniKani" title on the left side of your screen. The number on the lower righthand corner shows you how many lessons you currently have.
```
Comment (JenKay) [inline _posts/2018-10-31-first-lessons.md:12]: Can we replace "title" with the word "logo"?

**Pull request merged**